recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
I called it ampl-mp when building cross compiled Windows binaries on the opensuse build service. asl had been an existing formula in homebrew using the netlib source before the ampl/mp version moved to github. Are Debian and Fedora packaging libasl yet, and if so from which sources?